sdk/keyvault/keyvault-certificates/review/keyvault-certificates.review.api.md

```diff
@@ -0,0 +1,286 @@
+    maxresults?: number;
+}
+
+// @public
+export interface KeyVaultClientGetDeletedCertificatesOptionalParams extends coreHttp.RequestOptionsBase {
+    includePending?: boolean;
+    maxresults?: number;
+}
+
+// @public
+export interface KeyVaultClientImportCertificateOptionalParams extends coreHttp.RequestOptionsBase {
+    certificateAttributes?: CertificateAttributes;
+    // Warning: (ae-forgotten-export) The symbol "CertificatePolicy" needs to be exported by the entry point index.d.ts
+    certificatePolicy?: CertificatePolicy_2;
+    password?: string;
+    tags?: {
+        [propertyName: string]: string;
+    };
+}
+
+// @public
+export interface KeyVaultClientSetCertificateIssuerOptionalParams extends coreHttp.RequestOptionsBase {
+    attributes?: IssuerAttributes;
+    credentials?: IssuerCredentials;
+    organizationDetails?: OrganizationDetails;
+}
+
+// @public
+export interface KeyVaultClientUpdateCertificateIssuerOptionalParams extends coreHttp.RequestOptionsBase {
+    attributes?: IssuerAttributes;
+    credentials?: IssuerCredentials;
+    organizationDetails?: OrganizationDetails;
+    provider?: string;
+}
+
+// @public
+export interface KeyVaultClientUpdateCertificateOptionalParams extends coreHttp.RequestOptionsBase {
+    certificateAttributes?: CertificateAttributes;
+    certificatePolicy?: CertificatePolicy_2;
+    tags?: {
+        [propertyName: string]: string;
+    };
+}
+
+// @public
+export interface LifetimeAction {
+    // Warning: (ae-forgotten-export) The symbol "Action" needs to be exported by the entry point index.d.ts
+    action?: Action;
+    // Warning: (ae-forgotten-export) The symbol "Trigger" needs to be exported by the entry point index.d.ts
+    trigger?: Trigger;
+}
+
+// @public
+export interface NewPipelineOptions {
+    // (undocumented)
+    HTTPClient?: HttpClient;
+    // (undocumented)
+    logger?: HttpPipelineLogger;
+    // (undocumented)
+    proxyOptions?: ProxyOptions;
+    // (undocumented)
+    retryOptions?: RetryOptions;
+    telemetry?: TelemetryOptions;
+}
+
+// @public
+export interface OrganizationDetails {
+    // Warning: (ae-forgotten-export) The symbol "AdministratorDetails" needs to be exported by the entry point index.d.ts
+    adminDetails?: AdministratorDetails[];
+    id?: string;
+}
+
+// @public (undocumented)
+export interface ParsedKeyVaultEntityIdentifier {
+    name: string;
+    vaultUrl: string;
+    version?: string;
+}
+
+// @public
+export interface ProxyOptions {
+    // (undocumented)
+    proxySettings?: string;
+}
+
+// @public
+export interface RetryOptions {
+    readonly maxRetryDelayInMs?: number;
+    readonly retryCount?: number;
+    readonly retryIntervalInMS?: number;
+}
+
+// @public
+export interface SecretProperties {
+    contentType?: string;
+}
+
+// @public (undocumented)
+export interface TelemetryOptions {
+    // (undocumented)
+    value: string;
+}
+
+// @public
+export interface X509CertificateProperties {
```

Comment:
We got rid of this interface as well and moved it's properties up to CertificatePolicy